manager/apis/app/v1alpha1/m4dapplication_types.go

```diff
@@ -201,3 +201,9 @@ type M4DApplicationList struct {
 func init() {
 	SchemeBuilder.Register(&M4DApplication{}, &M4DApplicationList{})
 }
+
+const (
+	ApplicationClusterLabel = "app.m4d.ibm.com/app_cluster"
```

Comment:
For consistency with our other labels this should be `m4d.ibm.com/appCluster`. Same for the others.